config/dotfiles/gitconfig

```diff
@@ -1,49 +1,53 @@
 [alias]
-  s = status -sb
-  co = checkout
-  cleanup = !git-cleanup
-  cp = cherry-pick
-  last = log -1 HEAD
-  cm = commit
-  cma = commit --amend --reuse-message=HEAD
-  rb = rebase
-  rs = reset
-  p = push
-  pf = push --force-with-lease
-  pnv = push --no-verify
-  superclean = clean -fxd
-  # Rebase the current branch over origin/master
-  rbm = pull --rebase origin master
+	s = status -sb
+	co = checkout
+	cleanup = !git-cleanup
+	cp = cherry-pick
+	last = log -1 HEAD
+	cm = commit
+	cma = commit --amend --reuse-message=HEAD
+	rb = rebase
+	rs = reset
+	p = push
+	pf = push --force-with-lease
+	pnv = push --no-verify
+	superclean = clean -fxd
+	# Rebase the current branch over origin/master
+	rbm = pull --rebase origin master
 [color]
-  ui = auto
+	ui = auto
 [core]
-  excludesfile = ~/.gitignore_global
+	excludesfile = ~/.gitignore_global
 	precomposeunicode = true
 	ignorecase = false
-	untrackedCache = true
```

Comment:
This is set automatically by `feature.manyFiles` ([docs](https://git-scm.com/docs/git-config#Documentation/git-config.txt-featuremanyFiles)).